sagemaker-remote-function/quick_start/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: remote_conda_env
+channels:
+  - defaults
+dependencies:
+  - python=3.10
+  - pandas
+  - pip:
+      - sagemaker
```

Comment:
missing version?

Reply:
I don't think we need to specify the version here since the conda env will not use the sagemaker package installed at the system level. But adding anyways. :)